cfme/tests/configure/test_visual_infra.py

```diff
@@ -365,8 +375,45 @@ def test_template_noquads(appliance, set_template_quad):
         casecomponent: Configuration
         caseimportance: medium
         initialEstimate: 1/10h
+        tags: settings
     """
     view = navigate_to(appliance.collections.infra_templates, 'TemplatesOnly')
     view.toolbar.view_selector.select('Grid View')
     # Here data property will return an empty dict when the Quadrants option is deactivated.
     assert not view.entities.get_first_entity().data
+
+
+def test_change_truncate_long_text_save_button_enabled(appliance):
```

Comment:
**Optional**: maybe shorten this test case name?

Reply:
I think the test name reflects the case this test is testing, I'd like to keep it as is.